homeassistant/components/unifi/update.py

```diff
@@ -4,7 +4,7 @@
 from collections.abc import Callable, Coroutine
 from dataclasses import dataclass
 import logging
-from typing import TYPE_CHECKING, Any, Generic
```

Comment:
Was it intended to include the update platform changes in this PR?

Reply:
Yes, because the change in definition of DataT and HandlerT wasn't compatible with update